modules/helm-jhub/main.tf

```diff
@@ -67,12 +67,6 @@ module "shared-nfs" {
 
 locals {
   helm_release_wait_condition = length(kubernetes_secret.tls_secret) > 0 ? kubernetes_secret.tls_secret[0].metadata[0].name : kubernetes_namespace.jhub.metadata[0].name
```

Comment:
hm - why did this nfs code change? We also could check this deployment, which is not part of the tests

Reply:
Oh @mirestrepo actually tflint complained that this is declared by unused. It was used in line 220 of this file, which was commented out, so I just removed it. Should be safe to leave out then?